website/packages/docs/src/pages/css-extraction-webpack.mdx

```diff
@@ -99,46 +195,46 @@ Note that we don't currently support `style-loader` for loading `css` files from
 +const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
 module.exports = {
-  module: {
-    rules: [
-      {
-        test: /\.(js|ts|tsx)$/,
-        exclude: /node_modules/,
-        use: [
-          { loader: 'babel-loader' },
-          {
-            loader: '@compiled/webpack-loader',
-            options: {
-              extract: true,
-            },
-          },
-        ],
-      },
-+      {
-+        test: /\.compiled-css.css$/i,
-+        use: [MiniCssExtractPlugin.loader, 'css-loader'],
-+      },
-      // The following loader will be used for css files that are not from Compiled.
-      {
--        test: /\.css$/i,
-+        test: /(?<!compiled-css)(?<!\.compiled)\.css$/,
-        // Put your existing css loader configuration here
-        use: ['style-loader', 'css-loader'],
-      },
-    ],
-  },
-  plugins: [
-+    new MiniCssExtractPlugin(),
-    new CompiledExtractPlugin(),
-  ],
+	module: {
+		rules: [
+			{
+				test: /\.(js|ts|tsx)$/,
+				exclude: /node_modules/,
+				use: [
+					{ loader: 'babel-loader' },
+					{
+						loader: '@compiled/webpack-loader',
+						options: {
+							extract: true,
+						},
+					},
+				],
+			},
++			{
++				test: /\.compiled-css.css$/i,
```

Comment:
Not 100% certain on which suggestion it is actually, but isn't it this?
```suggestion
+				test: /^compiled-css\.css$/i,
```
If not, this:
```suggestion
+				test: /\.compiled-css\.css$/i,
```

Reply:
oops you're right, forgot to update the webpack part

Reply:
Yeah, sounds good.